src/libraries/Microsoft.PowerFx.Core/Logging/StructuralPrint.cs

```diff
@@ -259,6 +259,11 @@ public override LazyList<string> Visit(VariadicOpNode node, Precedence parentPre
             }
         }
 
+        public override LazyList<string> Visit(StrInterpNode node, Precedence context)
+        {
+            throw new NotImplementedException();
```

Comment:
couldn't this be basically, write "$"", then for each child, if it's a string literal write the string literal, if it's not, write `{` node.Accept() `}`, then write the final "?

Reply:
basicall, this'll need to be implemented before enabling, it's called from canvas.

Reply:
Added an implementation. Do we have tests for this in PowerFx or are they only in PA Client?